tests/scripts/all.sh

```diff
@@ -1855,6 +1865,16 @@ component_test_valgrind_constant_flow () {
     # details are left in Testing/<date>/DynamicAnalysis.xml
     msg "test: some suites (full minus MBEDTLS_USE_PSA_CRYPTO, valgrind + constant flow)"
     make memcheck
+
+    # Test asm path in constant time module - by default, it will test the plain C
+    # path under Valgrind or Memsan. Running only the constant_time tests is fast (<1s)
+    msg "test: valgrind asm constant_time"
+    scripts/config.py --force set MBEDTLS_TEST_CONSTANT_FLOW_ASM
+    skip_all_except_given_suite test_suite_constant_time
```

Comment:
Is there no other way to run a single suite, except by omitting all the others? That seems a bit strange

Reply:
AFAICT, no. This ends up going via `tests/scripts/run-test-suites.pl`, which offers a "--skip" parameter but not a "--only-run" parameter.

Reply:
`run-test-suites.pl` is only used with make, not with cmake. With cmake, running the tests uses the `ctest` utility that ships with cmake. Both methods have a skip ability because that's intended to skip flaky or slow tests. If you want to positively enumerate the test suites to run, you can manually run them one by one.

Valgrind is a bit of a special case because it doesn't run the executables directly: it runs them via `valgrind` with a bunch of options and then checks a log file to see if there were any problems. We rely on `ctest` to invoke valgrind and check the logs. I don't know how to run a single test suite with valgrind (except by duplicating what ctest does, which I've done in `mbedtls-prepare-build` but this is not present in the mbedtls repository).